robot/Cumulus/resources/NPSP.py

```diff
@@ -1166,13 +1171,134 @@ def click_wrapper_related_list_button(self,heading,button_title):
         """  
         locator = npsp_lex_locators["record"]["related"]["button"].format(heading, button_title)
         element = self.selenium.driver.find_element_by_xpath(locator)
-        self.selenium.driver.execute_script('arguments[0].click()', element)   
+        self.selenium.driver.execute_script('arguments[0].click()', element)  
+
+# Keywords below this line are for accessibility testing 
+
+
+
+    def inject_axe_core_library(self):
+        path = os.path.dirname(os.path.abspath(__file__))
+        self.selenium.execute_javascript(path + "/axe.js")
+    
+    
+    def get_axe_analysis_results(self,cssSelector = None):    
+        context = "document"
```

Comment:
It's possible to call [axe.run(...)](https://www.deque.com/axe/axe-for-web/documentation/api-documentation/#api-name-axerun) on a specific HTML DOM Element.  I see here this method checks the whole `document`.   It's possible there could be axe violations in parts of the `document` that a developer cannot control, e.g. Salesforce header bar.

Is it possible to add a robot method where `axe.run` could be called for a specified element such as through a given CSS selector?